components/cmumps2fhir.js

```diff
@@ -101,9 +101,9 @@ function graphUri(translation) {
 
         // TODO: Replace the hard coded patientID with a metadata key
         if (_.isUndefined(state.patientId)) {
-            uri = 'urn:local:' + encodeURIComponent(translator) + ':' + resourceId;
+            uri = 'urn:local:fhir:' + encodeURIComponent(translator) + ':' + resourceId;
         } else {
-            uri = 'urn:local:' + state.patientId + ':' + encodeURIComponent(translator) + ':' + resourceId;
+            uri = 'urn:local:fhir:' + state.patientId + ':' + encodeURIComponent(translator) + ':' + resourceId;
```

Comment:
1. I didn't think about this before, but we should probably call encodeURIComponent on the patientId and resourceId too, for safety.

2. If there is no patientId, I would rather still have that field in the URN retained but empty, so that it can be consistently parsed, like urn:local::myTranslator:1234